docs/advanced-usage/clerk-idp.mdx

```diff
@@ -4,11 +4,19 @@ description: Learn how to use Clerk to facilitate Single Sign-On (SSO) with othe
 ---
 
 > [!WARNING]
-> This feature will not return a Clerk JWT and does not provide authentication. Please see the [FAQ](#how-can-the-access-tokens-obtained-from-the-token-url-be-used) below for more information.
+> **This feature is not designed for handling authentication directly in your application.** To handle authentication _in your_ application, you can [configure one of the many OAuth providers that Clerk offers](/docs/authentication/social-connections/oauth#configuration), such as Google.
 
-Clerk can be configured as an identity provider to facilitate Single Sign-On (SSO) with other clients that support the OAuth 2.0 protocol. With this feature, your users can sign in with your Clerk application on other websites to authorize sharing of their user info.
+Clerk can be configured as an identity provider to facilitate Single Sign-On (SSO) with other clients that support the OAuth 2.0 protocol. This feature allows users to sign in to other applications using their Clerk credentials, enabling user information sharing between your Clerk application and external websites.
 
-![Clerk IDP Flow](/docs/images/advance/clerk-idp-flow.svg)
+## When should you use Clerk as an OAuth provider?
+
+You can use Clerk as an OAuth provider if you want your users to sign in to a third party site or a tool (e.g., Zapier or OpenAI) using their credentials from your application. **This is not the same as supporting an OAuth provider, such as Google, in your application. If you want your users to be able to sign in to your application with an OAuth provider, see [the dedicated guide](/docs/authentication/social-connections/oauth#configuration).**
```

Comment:
Small edit here - I'd be hesitant to recommend Zapier/OpenAI before we actually know it works with them though 😅

```suggestion
You can use Clerk as an OAuth provider if you want your users to sign in to a third party site or a tool (e.g., [Zapier](https://zapier.com/), [OpenAI](https://openai.com/), [Discourse](https://www.discourse.org/)) using their credentials from one of your existing Clerk applications. **This is not the same as supporting an OAuth provider, such as Google, in your application. If you want your users to be able to sign in to your application with an OAuth provider, see [the dedicated guide](/docs/authentication/social-connections/oauth#configuration).**
```

Reply:
OpenAI has worked for some customers, though I don't know the exact config they used. I am pretty sure Zapier has as well, but again I haven't tried. Maybe we use '3rd party tools' for the now until @jeremy-clerk has Zapier working and then mention it?